libs/rdf-tensor/src/dice/rdf-tensor/RDFNodeHashes.hpp

```diff
@@ -5,13 +5,6 @@
 #include <rdf4cpp/rdf.hpp>
 
 namespace dice::hash {
-	template<typename Policy>
```

Comment:
Why did you remove the hash for Node?

Reply:
already exists in rdfcpp in this version